src/OpenDiffix.Core.Tests/QueryEngineProperty.Tests.fs

```diff
@@ -50,7 +50,7 @@ let removeAt list index =
 // Other clauses' specifications
 // ----------------------------------------------------------------
 
-type LimitClauseSpec = NonNegativeInt option
+type LimitClauseSpec = { Value: NonNegativeInt; Include: bool }
```

Comment:
The `LIMIT` clause is handled after anonymization, there is no value in running tests for it, I would drop it completely instead.